interoperator/pkg/webhook/server/sfserviceinstance/mutating/sfserviceinstance_create_update_handler.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2018 The Service Fabrik Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mutating
+
+import (
+	"context"
+	"net/http"
+
+	osbv1alpha1 "github.com/cloudfoundry-incubator/service-fabrik-broker/interoperator/pkg/apis/osb/v1alpha1"
+	yaml "gopkg.in/yaml.v2"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/inject"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission/types"
+)
+
+func init() {
+	webhookName := "mutating-create-update-sfserviceinstance"
+	if HandlerMap[webhookName] == nil {
```

Comment:
Why is this condition needed?

Reply:
In the current scenario `HandlerMap[webhookName]` will always be empty when `init` is called for package `mutating`. I guess we can remove this check, although I'm skeptical since it is only an extra safety measure for dealing with a globally accessible map. Hypothetically - someone might accidentally register other handlers corresponding to `webhookName` elsewhere (maybe in `server` package) and we'll end up losing them when `init` resets the map to empty. Let me know if you think otherwise.